5_databases/week2/part4.sql

```diff
@@ -0,0 +1,43 @@
+USE farm;
```

Comment:
This implementation does not reflect your ERD diagram :sweat_smile: 
the issue is your many to many relationship are not represented correctly.
farmer and dairyproducts is a many to may relationship which means you are going to need a farm_dairyproduct table. so the rule is one to one, one to many and many to one can be represented in the same table. many to many implies a third table to hold the relation.
This applies also to all the many to many relationship you defined in the ERD Diagram